packages-exp/auth-exp/src/core/strategies/custom_token.ts

```diff
@@ -56,3 +60,30 @@ export async function signInWithCustomToken(
   await authInternal._updateCurrentUser(cred.user);
   return cred;
 }
+
+/**
+ * Sets a custom token provider, which is invoked when a new Firebase ID token is requested and no
+ * refresh token is present (i.e., in passthrough mode). This callback should be implemented to
+ * obtain a custom token to exchange for a new Firebase ID token.
+ *
+ * @param auth - The Auth instance.
```

Comment:
Could you add a short snippet example here? For example:
https://github.com/firebase/firebase-js-sdk/blob/f825b1d83228747e404aaefd9f3948d12257d0fc/packages-exp/auth-exp/src/core/auth/initialize.ts#L44-L48

Reply:
Cool, added the one [go/firebase-auth-passthrough-sdk-api#heading=h.4ej4y6o7kzi5](http://go/firebase-auth-passthrough-sdk-api#heading=h.4ej4y6o7kzi5)